yoti/admin/views/yoti/tmpl/default.php

```diff
@@ -8,14 +8,15 @@
 <div class="row-fluid">
     <div class="span12">
         <form id="yoti-form" method="post" enctype="multipart/form-data" action="<?php echo $_SERVER['REQUEST_URI']; ?>" class="form-horizontal">
+            <?php echo JHTML::_('form.token'); ?>
             <div class="alert">
             <p>
-                <?php echo JText::_('You need to first create a Yoti App at'); ?> <a href="<?php echo \Yoti\YotiClient::DASHBOARD_URL; ?>" target="_blank">
-                    <?php echo JText::_('Yoti Dashboard'); ?>
+                <?php echo JText::_('You need to first create a Yoti App at'); ?> <a href="<?php echo YotiHelper::YOTI_HUB_URL; ?>" target="_blank">
+                    <?php echo JText::_('Yoti Hub'); ?>
```

Comment:
Dashboard->Hub should probably be in the release notes?

Reply:
Updated PR description